labs/authentication-passport/README.md

```diff
@@ -80,6 +91,37 @@ app
   });
 ```
 
+- 4. Decorate your endpoint
+
+To authenticate your request with the basic strategy, decorate your controller
+function like:
+
+```ts
+import {AUTH_STRATEGY_NAME} from './my-basic-auth-strategy';
```

Comment:
nitpick: it seems like we assume `my-basic-auth-strategy` file is in `src/controllers` here and `src/` from the previous statement in the `application.ts` file. can we have it at a consistent place like `src/`?

Reply:
@b-admike I don't want to have a strong implication of the "right" file structure for users, they should decide it, that's why I just leave all the file import as `./file_name`.